### ISSUE TITLE
High CPU/RAM usage

### ISSUE THREAD

**Issue by Keerp96** [File-New-Project/EarTrumpet#1424]
### Summary

EarTrumpet uses over 50% of CPU while system is idle.
![Screenshot_2](https://github.com/File-New-Project/EarTrumpet/assets/1152200/fa6b4ca8-3f6e-4303-a563-ad92568715f5)

### Steps to reproduce

Have EarTrumpet running in the background.

### EarTrumpet version

2.2.2.0

### Windows version

10.0.19045.2965

### Additional information

Memory dump - https://pixeldrain.com/u/i66fa8RM

**Comment by riverar** [comment]
Taking a look. You can delete that dump too.

**Comment by riverar** [comment]
@Keerp96 Does restarting EarTrumpet bring CPU usage back down?

**Comment by Hyperseeker** [comment]
Having the same issue, though my instance uses 25% of CPU (which would be relative anyway) and less than 100 MB of RAM periodically. (Just now it sat at 78 MB, while idling at 0% CPU usage with 68 MB.) I can tell something's happening because there's a spike in my laptop's fan speed.

Restarting fixes the issue. The CPU usage also eventually disappears on its own. It almost sounds like its trying to check for updates *very* inefficiently or something: it happens rarely enough where it's not ordinarily an issue.

Any way I can assist in solving this?

EarTrumpet v2.2.2.0, Windows v10.0.19045.3208

**Comment by Keerp96** [comment]
> @Keerp96 Does restarting EarTrumpet bring CPU usage back down?

Yes, and as Hyperseeker mentioned it does seem to go away eventually on it's own.
